src/cmd/csi/init/builder.go

```diff
@@ -56,7 +57,7 @@ func (builder CommandBuilder) buildRun() func(*cobra.Command, []string) error {
 
 		csiManager, err := createManager(builder.namespace, kubeConfig)
 		if err != nil {
```

Comment:
Can't we make that error check more explicit than just allowing any error?

Reply:
not really there is not special error that is returned, and I would like to avoid checking the message and/or StatusCode as they can be fickle.